pkg/feeds/maven/maven.go

```diff
@@ -0,0 +1,141 @@
+package maven
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/ossf/package-feeds/pkg/feeds"
+)
+
+const (
+	FeedName  = "maven"
```

Comment:
This is a Maven Central feed, is it possible that we may want other Maven repository feeds in the future?

How would we distinguish those?

Reply:
Actually from Sonatype we can get more feeds apart from Maven. We are just filtering by repo name here. So it can be that we may have different feeds using same engine, but picking different repos. How do you see that, duplicating the feed and giving a new name?